hal/src/uhura/include.mk

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Let's not use product code-names. Tron is a leftover from the codebase cleanup. Anything related to trackerm should be trackerm.